src/modules/keyboardmanager/KeyboardManagerEditorLibrary/KeyDropDownControl.h

```diff
@@ -58,7 +63,8 @@ class KeyDropDownControl
 
 public:
     // Pointer to the keyboard manager state
-    static KeyboardManagerState* keyboardManagerState;
+    static KBMEditor::KeyboardManagerState* keyboardManagerState;
+    static KeyboardManagerShortcuts* keyboardManagerShortcuts;
```

Comment:
It's absurd that we have `keyboardManagerState` inside of `KeyDropDownControl`. But let's keep it this way as we are going to rewrite the editor completely.

Reply:
I agree, let's make the changes required to decouple the Editor from the Engine only. For the Editor refactoring it might make more sense to postpone it to the new UI.